src/herbpy/herbrobot.py

```diff
@@ -173,10 +173,22 @@ def __init__(self, left_arm_sim, right_arm_sim, right_ft_sim,
         import herbpy.action
         import herbpy.tsr
 
+        # Initialize herbpy ROS Node
+        import rospy
+        if not rospy.core.is_initialized():
+            rospy.init_node('herbpy', anonymous=True)
+            logger.debug('Started ROS node with name "%s".', rospy.get_name())
+
```

Comment:
Please move this back into the `if not self.talker_simulated` block. Starting a ROS node is only required if you want to use the talker service, so there's no point in spinning on up by default.